app/components/custom/budgets/investments/form_component.rb

```diff
@@ -0,0 +1,3 @@
+class Budgets::Investments::FormComponent < ApplicationComponent; end
+
+require_dependency Rails.root.join("app", "components", "budgets", "investments", "form_component").to_s
```

Comment:
I know it isn't necessary, but to make it clear that custom code must be added below this line, we could write:

```suggestion
class Budgets::Investments::FormComponent < ApplicationComponent; end

require_dependency Rails.root.join("app", "components", "budgets", "investments", "form_component").to_s

class Budgets::Investments::FormComponent 
end
```